docs/resources/aws_s3_bucket.md.erb

```diff
@@ -58,6 +58,12 @@ The following examples show how to use this InSpec audit resource.
     describe aws_s3_bucket('test_bucket') do
       it { should_not be_public }
     end
+
+### Check if a bucket tag exists
+
+    describe aws_ec2_instance('i-090c29e4f4c165b74') do
```

Comment:
Copypasta - should be aws_s3_bucket

Reply:
Fixed.  Also added an additional example and moved it to the properties section to address your other comment